libtelemetry/src/main/java/com/mapbox/android/telemetry/TelemetryClient.java

```diff
@@ -50,6 +53,40 @@ void sendEvents(List<Event> events, Callback callback) {
     sendBatch(batch, callback);
   }
 
+  void sendAttachment(Attachment attachment, Callback callback) {
+    List<VisionAttachment> visionAttachments = attachment.getAttachments();
+    List<AttachmentMetadata> metadataList = new ArrayList<>();
+
+    MultipartBody.Builder requestBodyBuilder = new MultipartBody.Builder("--01ead4a5-7a67-4703-ad02-589886e00923")
```

Comment:
I wonder why this value is hardcoded and if there are any downsides to hardcoding it?